module/os/linux/zfs/zvol_os.c

```diff
@@ -130,9 +266,11 @@ zvol_write(zv_request_t *zvr)
 	ssize_t start_resid = uio.uio_resid;
 	unsigned long start_time;
 
+#ifndef HAVE_BLK_MQ
```

Comment:
```suggestion
/*
 * When HAVE_BLK_MQ is defined accounting is done by blk_mq_start_request()
 * and blk_mq_end_request() in the zvol_mq_work_func() function.
 */
#ifndef HAVE_BLK_MQ
```

Or better yet, it looks like you could pull this accounting code out of `zvol_write()` and `zvol_read()` and move it to the non-mq `zvol_request`.  This would then match what is being measure in the multi-queue case.